src/components/shared/thread/ActivityGraph.js

```diff
@@ -103,14 +113,21 @@ class ThreadActivityGraph extends PureComponent<ActivityGraphProps> {
     canvas.width = canvasPixelWidth;
     canvas.height = canvasPixelHeight;
 
-    this._activityGraphFills = new ActivityGraphFills(
+    const { fills, fillsQuerier } = computeActivityGraphFills({
       canvasPixelWidth,
       canvasPixelHeight,
-      this.props,
-      this._getCategoryDrawStyles(ctx)
-    );
+      fullThread,
+      interval,
+      rangeStart,
+      rangeEnd,
+      samplesSelectedStates,
+      xPixelsPerMs: canvasPixelWidth / (rangeEnd - rangeStart),
+      treeOrderSampleComparator: treeOrderSampleComparator,
```

Comment:
I would have thought we'd have a lint for this. `a: a,` -> `a,`

Reply:
I thought I had already added a lint for this, odd.